Jenkinsfile

```diff
@@ -50,17 +50,15 @@ pipeline {
     rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
   }
   triggers {
-    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*')
+    issueCommentTrigger('(?i).*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests\\W+(?:for\\W+macos)?(?:\\W+please)?.*')
```

Comment:
Could we simplify the message to `/test` to run the job, and `/test macos` or `/test macos windows` to run another tests

```suggestion
    issueCommentTrigger('(?i)(.*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*|^\/test(\W+macos)?(\W+windows)?$)')
```

Reply:
even do we keep the `jenkins run the test please` message

Reply:
shit I trigger a build with that, because of that I don't like it.

Reply:
I'll update the README.md accordingly then and commit all together

Reply:
The comment `/test windows` is not supported at the moment